src/components/DataDictionary.tsx

```diff
@@ -90,49 +91,59 @@ export const columns: ColumnDescriptor[] = [
     name: "category",
     label: "Category",
     rowValueDescriptor: "category",
-    formatter: (x: any, isRow: boolean, field: ILookmlModelExploreField) => {
-      return <CategorizedLabel label={x} category={field.category} />
-    },
+    formatter: Object.assign(
+      ({ x, field }: FormatterProps) => {
+        return <CategorizedLabel label={x} category={field.category} />
+      },
+      { displayName: "CategoryLabel" }
+    ),
```

Comment:
I found this cool but hard-to-read way to get around the `react/display-name` lint error. 

I didn't want to just disable the error because I don't see that configuration in sdk-codegen or components.

I also don't think these formatter funcs deserve their own files, but perhaps they do. LMK if you'd like me to break them into their own files with the `export default` that satisfies the `display-name` req't, or if this is OK....

Reply:
One of many sources about this https://medium.com/@stevemao/do-not-use-anonymous-functions-to-construct-react-functional-components-c5408ec8f4c7

Reply:
I think `function CategoryLabel () {}` is best here, as you had it before. What I don't understand is what it isn't a problem with the other elements in this array.